cfme/tests/services/test_dialog_element_in_catalog.py

```diff
@@ -115,3 +116,658 @@ def test_validate_not_required_dialog_element():
+        initialEstimate: 1/6h
+        testSteps:
+            1. Create a service catalog with type orchestration template (heat or Aws)
+            2. Order the above service from global region
+        expectedResults:
+            1.
+            2. From global region, the ordering of catalog should be successful
+
+    """
+    pass
+
+
+@pytest.mark.meta(coverage=[1713100])
+@pytest.mark.manual
+@pytest.mark.tier(2)
+def test_copy_save_service_dialog_with_the_same_name():
+    """
+
+    Bugzilla:
+        1713100
+
+    Polarion:
+        assignee: nansari
+        startsin: 5.10
+        casecomponent: Services
+        initialEstimate: 1/6h
+        testSteps:
+            1. Copy service dialog "Test Dialog" save as "Copy of Test Dialog"
+            2. Copy service dialog "Test Dialog" attempt to save as "Copy of Test Dialog"
+        expectedResults:
+            1.
+            2. Tabs shouldn't be copied and shouldn't show multiple times.
+
+    """
+    pass
+
+
+@pytest.mark.meta(coverage=[1696697])
+@pytest.mark.manual
+@pytest.mark.tier(2)
+def test_request_details_page_tagcontrol_field():
+    """
+
+    Bugzilla:
+        1696697
+
+    Polarion:
+        assignee: nansari
+        startsin: 5.10
+        casecomponent: Services
+        initialEstimate: 1/6h
+        testSteps:
+            1. Add service dialog with tagcontrol field
+            2. Add catalog item with above dialog
+            3. Order the service
+            4. Go to service request details page
+        expectedResults:
+            1.
+            2.
+            3.
+            4. No error when go to on service request details page
+
+    """
+    pass
+
+
+@pytest.mark.meta(coverage=[1694737])
+@pytest.mark.manual
+@pytest.mark.tier(2)
+def test_regex_validation_should_work():
+    """
+
+    Bugzilla:
+        1694737
+
+    Polarion:
+        assignee: nansari
+        startsin: 5.10
+        casecomponent: Services
+        initialEstimate: 1/6h
+        testSteps:
+            1. Import DataStore and Dynamic Dialog
+            2. Add catalog item with above dialog
+            3. Order the service
+        expectedResults:
+            1.
+            2.
+            3. Regex validation should work
+    """
+    pass
+
+
+@pytest.mark.meta(coverage=[1696474])
+@pytest.mark.manual
+@pytest.mark.tier(2)
+def test_dynamic_dialogs():
+    """
+
+    Bugzilla:
+        1696474
+
+    Polarion:
+        assignee: nansari
+        startsin: 5.10
+        casecomponent: Services
+        initialEstimate: 1/6h
+        testSteps:
+            1. Import DataStore and Dynamic Dialog
+            2. Add catalog item with above dialog
+            3. Order the service
+        expectedResults:
+            1.
+            2.
+            3. Dynamic dialog all functionality should work
+    """
+    pass
+
+
+@pytest.mark.meta(coverage=[1706600])
+@pytest.mark.manual
+@pytest.mark.tier(2)
+def test_dynamic_dialogs_on_service_request():
```

Comment:
Can these tests with the same `testSteps` be collated into a single test case? Also, it is unclear just from the testSteps what the test is testing, though I suppose the BZ will give some more information.